ebms-provider/src/test/kotlin/no/nav/emottak/ebms/EbmsRouteAsyncIT.kt

```diff
@@ -23,15 +23,16 @@ import no.nav.emottak.message.model.AsyncPayload
 import no.nav.emottak.message.xml.xmlMarshaller
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
 import org.xmlsoap.schemas.soap.envelope.Envelope
-import java.util.*
 import kotlin.uuid.ExperimentalUuidApi
 import kotlin.uuid.Uuid
 
 class EbmsRouteAsyncIT : EbmsRoutFellesIT("/ebms/async") {
 
     @Test
+    @Disabled
```

Comment:
Hvorfor slå disse av? Feiler disse? Er dette noe vi skal fikse senere?

Reply:
Disse testene er for et asynkront http-endepunkt, som er fjernet i sin nåværende tilstand. Det betyr at de feiler slik de er nå, men lar testene ligge foreløpig slik at de eventuelt kan omskrives og gjenbrukes til den nye flyten.